tests/src/CoreMangLib/system/highperformancecounter/highperformancecounter.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This is very weak test (e.g. compared to how we test stop watch). I do not think you need to bother to a test like this.

Reply:
The API basically exposes `QueryPerformanceCounter` and `QueryPerformanceFequency`.
I am just testing that they are connected. - not crashing with missing methods, etc...

Reply:
There is not a lot of functionality. I just could not do it without minimal testing to see that API works at all.

Reply:
The longer team value of such testing is super low. You will get the same level of testing by any use of this.

Reply:
I figured - if I didn't have tests, I'd be asked "why not a single test was added? how do we know it works?". I myself would feel uncomfortable. 
Removing a test is easy though.